packages/templates/distro.yaml

```diff
@@ -1,4 +1,6 @@
 {{ define "distro_install" }}
+## All packages for teal are in the teal image already.
+{{ if ne .Values.codename "teal" }}
```

Comment:
This is gating accidental calls to `zypper in..` if we are in `teal`. supposedly, we don't add any repo to it